assemblies/pmr-libraries/pom.xml

```diff
@@ -923,47 +923,38 @@
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-continuation</artifactId>
-      <version>8.1.15.v20140411</version>
```

Comment:
Should these hard-coded versions be put into a pom parameter instead?

Reply:
That's a removal, so it'll use the one that exists on maven-parent-pom (currently 8.1.15.v20140411 but will go to 9.4.x in the Karaf upgrade).
Do you need the pmr-libraries to have a fixed/overrided version also?